submissions/mod-comparison/mod-comparison.md

```diff
@@ -0,0 +1,107 @@
+
+AngelBobs is probably the weakest of the three. The new buildings certainly aren't ugly - it's generally clear what they do and they're mostly distinctive, however they don't quite fit in quite as well with the general Factorio diesel punk aesthetic as the others do. I also feel that sometimes they look pasted on, like a bad Photoshop - I think this is probably down to detail level, or lighting. Generally, different tiers are distinguished by a palette swap - you go from grey belts to yellow, red, blue, purple and green, which is similar to vanilla. There's some variety in the quality of the icons for the different materials, with some icons being very high resolution and others being very low. I suspect this comes from the graphics being gradually added over time or from different sources.. Most of the time, this doesn't matter, but sometimes you zoom in and the varied resolutions can really stick out. Some of the machines tiers are very difficult to tell apart - one of the crushers (I think) replaces a grey plate on the roof with a dark blue one and unfortunately the two look very similar. That said, I have seen mods with much much worse artwork than this and I certainly wasn't bothered by the quality of it while I was playing.
+
+![A smelting area](angelbobs.png)
+
+Space Exploration uses a lot of very large machines - especially in space - and these are all excellently rendered. I feel like they fit in very nicely with the general aesthetic, being a higher tech version but still feeling like they've come from the same family of machines. They're all nicely animated too, which makes it easy to tell how busy an area of the factory is. The consistent feel, style and resolution really helps everything come together nicely. That said, there is definitely some artwork reuse in there with a number of machines being recoloured versions of other ones. This doesn't matter as such - they're easy to tell apart and it's not confusing - however it is a minor downside. I've never struggled to tell buildings or resources apart, and only ever struggled to find them due to the sheer quantity of content.
+
+![Making ion stream](ionstream.png)
+
+I think that Industrial Revolution is the winner in the graphics area. It doesn't beat out Space Exploration by much, but the visual interest offered by the three stages of technology - burner, electrical, and laser - really comes across in the artwork. Whilst the different tiers are distinguished by colour, there's much more to it than that. The assembly machines move from showing big cutting wheels to arc welding to visible laser beams. The furnaces are all completely different shapes. In short, it feels like the machines are all completely distinct, rather than being repainted versions of the tier below. In short, it feels like a lot of care and attention has gone into the artwork in this mod, and the mod looks very good because of it.
```

Comment:
```suggestion
I think that Industrial Revolution is the winner in the graphics area. It doesn't beat out Space Exploration by much, but the visual interest offered by the three stages of technology - burner, electrical, and laser - really comes across in the artwork. Whilst the different tiers are distinguished by colour, there's much more to it than that. The assembly machines move from showing big cutting wheels to arc welding to visible laser beams. The furnaces are all completely different shapes. It feels like the machines are all completely distinct, rather than being repainted versions of the tier below. In short, it feels like a lot of care and attention has gone into the artwork in this mod, and the mod looks very good because of it.
```